src/service/run-api-service.js

```diff
@@ -494,6 +495,11 @@ module.exports = function (config) {
                 runService: this
             }));
             return vs;
+        },
+
+        introspection: function (config) {
```

Comment:
Discussed with @narenranjit, it makes sense to have it here similar to variables(). I would just additionally expose it in the app.js file.